docs/public/lib/use-localstorage.js

```diff
@@ -0,0 +1,18 @@
+import { useState } from 'preact/hooks';
+
+/**
+ * @type {<T>(name: string, value: T) => [T, (v: T) => void]}
+ */
+export function useLocalStorage(key, initial) {
+	const [v, setValue] = useState(() => {
+		const stored = localStorage.getItem(key);
```

Comment:
Hmm, does this work with pre-rendering? The error in netlify refers to this being undefined